webapp/components/wpt-metadata.js

```diff
@@ -221,6 +225,13 @@ class WPTMetadata extends LoadingState(PolymerElement) {
     this.shadowRoot.querySelector('#metadata-toggle').hidden = true;
     this.shadowRoot.querySelector('#metadata-collapsible').opened = true;
   }
+
+  checkPath(testname, path) {
+    if (testname.endsWith('/*')) {
+      return path.startsWith(testname.substring(0, testname.length - 1));
```

Comment:
Does the path always end with a trailing slash?

Reply:
Good catch! 

I don't think it ends with a trailing slash at all. I use `testname.length -1` to avoid the case when path is /foobar but testname is /foo/*. Added one more check